.github/workflows/check.yml

```diff
@@ -36,53 +37,53 @@ jobs:
           --features=${{ matrix.feature }} \
           //...
 
-  build:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        flag:
-          - '--config=clang-format'
-          - '--config=clang-tidy'
-          - '--config=verbose-clang-tidy'
-          - '--compilation_mode=opt'
-        exclude:
-          - flag: ${{ github.event_name == 'pull_request' && '--config=verbose-clang-tidy' || 'dummy' }}
-
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: mount bazel cache
-      uses: actions/cache@v3
-      with:
-        path: "~/.cache/bazel"
-        key: bazel-build-${{ matrix.flag }}
-
-    - name: install libtinfo5
-      # clang tools link against this
-      run: sudo apt-get install -y libtinfo5
-
-    - run: |
-        bazel \
-          --bazelrc=.github/workflows/ci.bazelrc \
-          build \
-          ${{ matrix.flag }} \
-          //...
-
-
-  buildifier:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: mount bazel cache
-      uses: actions/cache@v3
-      with:
-        path: "~/.cache/bazel"
-        key: bazel-buildifier
-
-    - run: |
-        bazel \
-          --bazelrc=.github/workflows/ci.bazelrc \
-          run \
-          //:buildifier.check
+#  build:
```

Comment:
Re-enable or delete all this?

Reply:
Will do once I get the build to work in CI

Reply:
done